Tests/scripts/utils/log_util.py

```diff
@@ -96,3 +96,21 @@ def install_logging(log_file_name: str, include_process_name=False) -> str:
     logging.basicConfig(level=logging.DEBUG,
                         handlers=[ch, fh])
     return log_file_path
+
+
+def install_simple_logging():
```

Comment:
Because I don't want to write to a logging file yet until it's fully implemented
And because I don't want to change the current behavior.

This is only to allow me to replace the print_error, print_warning, print_color methods with this one without any changes to the output.